README.md

```diff
@@ -2,96 +2,78 @@
 
 ## Benchmark Running Instructions
 
-To run benchmarking, navigate to `6.830FinalProject/benchmark_joins`. From there, you can run 
+To run benchmarking, navigate to `6.830FinalProject/benchmark_joins`. From there, you can run the `profiler.sh` script via, e.g.,:
 
-```cargo run [table1_name] [table2_name] [outfile_json]```
```

Comment:
cargo run is still a valid way to run a single experiment right? we should leave this documentation in?

Reply:
Yes, however the interface has changed. I include a blurb about the validity of cargo run with the new interface in the next commit